doc/source/_rulesets/main.json

```diff
@@ -0,0 +1,44 @@
+{
+  "id": 2045316,
+  "name": "Main branch protection",
+  "target": "branch",
+  "source_type": "Repository",
+  "source": "ansys-internal/pymechanical-meshworkflows",
```

Comment:
I do not think this is important but we could use a repo that is public.

Reply:
Please do... @AlejandroFernandezLuces remove this